packages/cbioportal-frontend-commons/src/components/downloadControls/DownloadControls.tsx

```diff
@@ -127,11 +128,22 @@ export default class DownloadControls extends React.Component<
                         }
                     });
                 } else {
-                    saveMethod(
-                        result,
-                        `${this.props.filename}.${fileExtension}`,
-                        { excludeCss: true }
+                    // using serlizer to convert svg to string
+                    let serializer = new XMLSerializer();
+                    const inlinedSVG = juice(
+                        serializer.serializeToString(result)
                     );
+
+                    // parsing as SVGElement after css inlining
+                    let parser = new DOMParser();
```

Comment:
how come we have to parse it back to SVG?  is it just because saveMethod takes SVG?

Reply:
yeah exactly **juice** returns a string containing inlined html and saveMethod takes only SVGElement